src/types.ts

```diff
@@ -1,3 +1,49 @@
 export interface ILocation {
-  location: Location;
+  location?: Location;
```

Comment:
The location was made optional in order for the build to work. 

The issue is that when placing pages in the `page` folder, `location` is made a available as a prop. 

This is so that the ScorecardTemplate would work.. see the comment in that file...